cleanenv.go

```diff
@@ -154,17 +154,17 @@ func parseFile(path string, cfg interface{}) error {
 }
 
 // parseYAML parses YAML from reader to data structure
```

Comment:
Please pay attention to proper Godoc comment names. It also makes since to extend comments since methods are published

Reply:
@ilyakaznacheev thanks for comment. Fixed